app/src/main/java/org/jellyfin/androidtv/integration/dream/composable/DreamContentNowPlaying.kt

```diff
@@ -116,7 +116,9 @@ fun DreamContentNowPlaying(
 				text = item.run {
 					if (!artists.isNullOrEmpty()) return@run artists?.joinToString(", ")
 					val albumArtistNames = albumArtists?.mapNotNull { it.name }
-					if (!albumArtistNames.isNullOrEmpty()) return@run albumArtistNames.joinToString(", ")
+					if (!albumArtistNames.isNullOrEmpty()) return@run albumArtistNames.joinToString(
+						", "
+					)
```

Comment:
This formatting is kind of weird... what about brackets for the `if` instead?

Reply:
Good catch, file was auto-formatted when I updated the import. I've rewritten the code here to make it more readable (while producing the same output).